cmd/globals.go

```diff
@@ -94,6 +95,8 @@ var (
 	// IsSSL indicates if the server is configured with SSL.
 	globalIsSSL bool
 
+	globalTLSCertificate *tls.Certificate
```

Comment:
I think `Srv` is a duplicate meaning as this certificate is used only in server and there is no client TLS certificate we have now.